modules/465-pod-reloader/templates/deployment.yaml

```diff
@@ -131,7 +130,7 @@ spec:
                     apiGroup: apps
                     apiVersion: v1
                     resource: deployments
-                    subresource: http
```

Comment:
🤔

Reply:
Only prometheus metrics are protected by kube-rbac-proxy.
We agreed to use `http` subresource if it protects a web interface, and `prometheus-metrics` for metrics.